src/System.Diagnostics.Debug/src/System/Diagnostics/DebuggerHiddenAttribute.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Diagnostics
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Constructor, Inherited = false)]
+    public sealed class DebuggerHiddenAttribute : Attribute
+    {
+        public DebuggerHiddenAttribute() { }
```

Comment:
I like having them because if we add another explicit constructor, we don't "lose" the implicit zero argument one.  I will keep them.